config/locales/en.yml

```diff
@@ -580,6 +580,10 @@ en:
     discovery: Discovery
     sensitive_content: Sensitive content
     toot_layout: Toot layout
+  notifications:
+    email_events: Events for e-mail notifications
+    email_events_hint: Select events that you want to receive notifications for
```

Comment:
This one clearly missing a termination! If you are reviewing this, please tell: should I use colon (`:`) here or just dot (`.`) is okay?